backend.native/tests/runtime/basic/random.kt

```diff
@@ -0,0 +1,95 @@
+package runtime.basic.random
+
+import konan.worker.*
+import kotlin.collections.*
+import kotlin.random.*
+import kotlin.system.*
+import kotlin.test.*
+
+private inline fun <reified T> getRandomNumbers(amount: Int = 10, producer: () -> T): List<T> {
```

Comment:
Not sure if this function got anything to do with random numbers, maybe smth like
`Array<T>(amount, producer).toList()` would be a shorter implementation.